deploy/docker/Dockerfile.web

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2023 OpenIM open source community. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build stage
+FROM node:18 AS builder 
+
+WORKDIR /openkf-web
+
+COPY /web .
+RUN npm install
+RUN chmod +x scripts/setup.sh && ./scripts/setup.sh
+RUN npm run build
+
```

Comment:
RUN npm install \
&& RUN chmod +x scripts/setup.sh && ./scripts/setup.sh \
&& RUN npm run build

no need to separate it to three layers

Reply:
Thanks!